model-datastructure/src/commonTest/kotlin/LinearHistoryTest.kt

```diff
@@ -57,133 +57,6 @@ class LinearHistoryTest {
         assertEquals(expected, actual)
     }
 
-    @Test
-    fun knownPerformanceIssue() {
-        // This test was dumped from actual case discovered during a profiling session.
-
-        val v30000003a = version(12884901946, null)
```

Comment:
If you mention the timeouts in the commit message, I would add that they were not caused by the production code, but by the expensive asserts in `assertHistoryIsCorrect`.

Someone might wonder why we did not look into why the test took so long.

Reply:
I've reworded the commit message to include this hint.